auth_server/server/server.go

```diff
@@ -38,6 +38,7 @@ import (
 
 var (
 	hostPortRegex = regexp.MustCompile(`\[?(.+?)\]?:\d+$`)
+	scopeRegex    = regexp.MustCompile("(\\w+)($|\\((\\w+)\\))")
```

Comment:
use backtick quoting to avoid "\\" stuttering.
use character class exactly as stated in the doc - `[a-z0-9]+`

Reply:
a-ha! Good catch. Fixed it! :)